.eslintrc.js

```diff
@@ -79,6 +79,9 @@ module.exports = {
     ],
     'jsdoc/check-alignment': 'off',
     'license-header/header': ['error', './license-header.js'],
+    'import/prefer-default-export': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
`unknown` seems to be way more restrictive than `any`, and it would require switching away from `unknown` and define new types. We might want to have another ticket to replace all `any` and `unkown` in the codebase.